app/css/pages/community-story.css

```diff
@@ -9,7 +9,7 @@ body.namespace-community.controller-stories.action-show {
 
     .back-link {
         @apply flex items-center relative py-32;
-        @apply text-textColor6 text-16 font-semibold leading-170;
+        @apply text-lightBlue text-16 font-semibold leading-170;
```

Comment:
IIRC think light blue on black violates accessibilty so I changed this. I think it should be even brighter really.

Reply:
I'm fine with that. then we can change the arrow's colour too, so it looks neat.